vsphere/datadog_checks/vsphere/config.py

```diff
@@ -50,7 +50,8 @@ def __init__(self, instance, log):
         self.metrics_per_query = instance.get("metrics_per_query", DEFAULT_METRICS_PER_QUERY)
         self.batch_collector_size = instance.get('batch_property_collector_size', DEFAULT_BATCH_COLLECTOR_SIZE)
         self.batch_tags_collector_size = instance.get('batch_tags_collector_size', DEFAULT_TAGS_COLLECTOR_SIZE)
-        self.should_collect_events = instance.get("collect_events", self.collection_type == 'realtime')
+        self.should_collect_events = instance.get("collect_events", self.collection_type in ['realtime', 'events_only'])
```

Comment:
We should start using constants here for `realtime` `historical` and `events_only`

Reply:
Not changed for now, can be done in another PR.